ArduCopter/ekf_check.cpp

```diff
@@ -123,7 +123,7 @@ bool Copter::ekf_over_threshold()
     }
 
     uint32_t now_us = AP_HAL::micros();
-    float dt = (now_us - last_ekf_check_us) / 1000000.0f;
+    float dt = (now_us - last_ekf_check_us) * 1e-6f;
```

Comment:
won't this bring much rounding issue as under 1e-5 ?

Reply:
IDK, will defer to @amilcarlucas and @rmackay9 who wanted this

Reply:
i don't see any rounding issues in this